lib/ansible/modules/database/postgresql/postgresql_vacuum.py

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'supported_by': 'community',
+    'status': ['preview']
+}
+
+DOCUMENTATION = r'''
+---
+module: postgresql_vacuum
+short_description: Garbage-collect and optionally analyze a PostgreSQL database
+description:
+- Garbage-collect and optionally analyze a PostgreSQL database
+  U(https://www.postgresql.org/docs/current/sql-vacuum.html).
+version_added: '2.9'
+
+options:
+  table:
```

Comment:
Multiple tables could be specified at once. For example, `vacuum (freeze, analyze, disable_page_skipping) table1, table2, table3 ;`

Reply:
thank you for this suggestion!
I thought about this but what to do with option 'columns' in this case?

Reply:
@kostiantyn-nemchenko , what do you think?

Reply:
I see multiple options here:

1)
```
- postgresql_vacuum:
    db: acme
    table: mytable1(col1, col2), mytable2(col1, col2)
    analyze: yes
```

2)
```
- postgresql_vacuum:
    db: acme
    table: mytable1, mytable2
    columns:
    - mytable1.col1
    - mytable1.col2
    - mytable2.col1
    analyze: yes
```

3) Simply issue an error when `columns` key is used with multiple tables.
    
4) Leave the current version of the module as is.

Personally, the 1st and the 2nd options look good to me. But if anyone has any objections, we should stick with the 3rd option at least.

Reply:
@kostiantyn-nemchenko , thank you for the reply!
1. The first way looks good to me but causes problems
```
table: mytable1(col1, col2), mytable2(col1, col2)  =>
"tables": [
    "mytable1(col1", 
    " col2)", 
    "mytable2(col1",
    " col2)"
]
```
pretty difficult to handle this list.. I tried, the code was becoming very complex
2. The second one, IMO, overloaded with repetition of table names (in this way, indeed, more simple to use postgresql_query: 'vacuum blabla (bla, bla)'
3. The third, IMO, is pretty castrated :)

Maybe the 4th ? That's simple :) I'd prefer the first one if there weren't the difficulties

Reply:
@kostiantyn-nemchenko Let's close this PR or leave it as it is now.
Otherwise, probably postgresql_query is the simplest and most flexible way to do vacuum